src/client/src/system/openFin/dockingManager.js

```diff
@@ -0,0 +1,843 @@
+// Source: https://github.com/openfin/snap-and-dock
```

Comment:
perhaps this would make more sense in a `lib` folder that sits beside `node_modules` then consume it via an `import`. Just makes it more obvious that it's not ours and we shouldn't touch it.

What do you think?